site/addons/Imgix/ImgixTags.php

```diff
@@ -21,17 +21,18 @@ protected function categorizedAttributes() {
 
         unset($attrs['path']);
 
-        while (list($key, $val) = each($attrs)) {
-            $is_html_attr = in_array($key, self::$html_attributes);
-            $is_data_attr = strpos($key, 'data-') === 0;
-            $is_aria_attr = strpos($key, 'aria-') === 0;
-
-            if ($is_html_attr || $is_data_attr || $is_aria_attr) {
-                $categorized_attrs['img_attributes'][$key] = $val;
-            } else {
-                $categorized_attrs['imgix_attributes'][$key] = $val;
-            }
-        }
+	    if (is_array($attrs)) {
+		    foreach ($attrs as $key => $val) {
+			    $is_html_attr = in_array($key, self::$html_attributes);
+			    $is_data_attr = strpos($key, 'data-') === 0;
+			    $is_aria_attr = strpos($key, 'aria-') === 0;
+			    if ($is_html_attr || $is_data_attr || $is_aria_attr) {
+				    $categorized_attrs['img_attributes'][$key] = $val;
```

Comment:
Looks like you've indented all these lines with a combination of tabs and spaces. Can we change it to match the spacing convention in the rest of the file?